client/extensions/woocommerce/state/sites/setup-choices/actions.js

```diff
@@ -40,3 +42,53 @@ export const fetchSetupChoices = ( siteId ) => ( dispatch, getState ) => {
 			dispatch( setError( siteId, getAction, err ) );
 		} );
 };
+
+const updateSetupChoice = ( dispatch, siteId, key, value ) => {
+	const postAction = {
+		type: WOOCOMMERCE_SETUP_CHOICE_UPDATE_REQUEST,
+		siteId,
+		key,
+		value
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/15223/commits/004863a3c5f48ce79a2d0ea93ba459112536c873